core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -503,6 +560,104 @@ public boolean namespaceExists(Namespace namespace) {
     return JdbcUtil.namespaceExists(catalogName, connections, namespace);
   }
 
+  @Override
+  public boolean dropView(TableIdentifier identifier) {
+    if (!viewSupport) {
+      LOG.warn(VIEW_WARNING_LOG_MESSAGE);
```

Comment:
this feels inconsistent with `newViewOps()`. Can we also throw a UOE here and in `listViews()` / `renameView()`?

Reply:
I was hesitating. But, yeah, that makes sense to throw `UnsupportedOperationException` here.